xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -175,7 +175,7 @@ void CPeripheralCecAdapter::Announce(AnnouncementFlag flag, const char *sender,
   else if (flag == GUI && !strcmp(sender, "xbmc") && !strcmp(message, "OnScreensaverActivated") && m_bIsReady)
   {
     // Don't put devices to standby if application is currently playing
-    if ((!g_application.m_pPlayer->IsPlaying() && !g_application.m_pPlayer->IsPaused()) && m_configuration.bPowerOffScreensaver == 1)
+    if (!g_application.m_pPlayer->IsPausedPlayback() && m_configuration.bPowerOffScreensaver == 1)
```

Comment:
> how you get "OnScreensaverActivated" if video playback is active (playing) and is not paused?

that is just an assumption. the code has to reflect what is written in the comment:
if ((!g_application.m_pPlayer->IsPlaying() && !g_application.m_pPlayer->IsPlaybackPaused()) ....